spec/log_parser_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "log_parser"
+describe LogParser do
+    context "call get_first_line" do
+        it "gets first line" do
+            log_parser = LogParser.new("spec/fixtures/game_test.log")
+            first_line = log_parser.get_first_line
+            expect(first_line).to eq("  0:00 ------------------------------------------------------------")
+        end
+    end
+
+    context "games.log does not exist" do
+        it "print file not found" do
+            log_parser = LogParser.new("game.log")
+            expect do
```

Comment:
depois de adicionar a verificação no initializer, utilize o expect exception do rspec: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher

Reply:
certo, não conhecia esse, obrigadinha